build/docker/centos7/build/Dockerfile

```diff
@@ -200,6 +200,14 @@ RUN source /opt/rh/devtoolset-8/enable && \
     cd .. && \
     rm -rf /tmp/*
 
+# download old fdbserver binaries
+ARG FDB_VERSION="6.2.29"
```

Comment:
for later PR, we may also want 6.3.9 fdb as the old binaries. This is to test the patch "upgrade", say from 6.3.9 to 6.3.12

Reply:
6.3.9 here broke communication with joshua coordinating cluster, I'll make that change in a separate PR.